### PR TITLE
8224617: (fs) java/nio/file/FileStore/Basic.java found filesystem twice

### PR DESCRIPTION
**Pull request by luchenlin**
I backport this for parity with 11.0.20-oracle.
No risk, only a test change.
Tests pass. SAP nightly testing passed.

test/jdk/java/nio/file/FileStore/Basic.java  
Copyright
test/lib/jdk/test/lib/util/FileUtils.java
Resolved due to context

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue

### Issue
 * [JDK-8224617](https://bugs.openjdk.org/browse/JDK-8224617): (fs) java/nio/file/FileStore/Basic.java found filesystem twice (**Bug** - P4)

### Reviewers
 * [Goetz Lindenmaier](https://openjdk.org/census#goetz) (@GoeLin - **Reviewer**) ⚠️ Review applies to [ec755845](https://git.openjdk.org/jdk11u-dev/pull/1933/files/ec7558458125910bc6c4377d20fac15f77b8908b)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jdk11u-dev.git pull/1933/head:pull/1933` \
`$ git checkout pull/1933`

Update a local copy of the PR: \
`$ git checkout pull/1933` \
`$ git pull https://git.openjdk.org/jdk11u-dev.git pull/1933/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 1933`

View PR using the GUI difftool: \
`$ git pr show -t 1933`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jdk11u-dev/pull/1933.diff">https://git.openjdk.org/jdk11u-dev/pull/1933.diff</a>

</details>

### Webrev
[Link to Webrev Comment](https://git.openjdk.org/jdk11u-dev/pull/1933#issuecomment-1578070570)

### CODE REVIEW

File: test/lib/jdk/test/lib/util/FileUtils.java [hunk 1]
```diff
@@ -238,6 +242,93 @@ public static boolean areFileSystemsAccessible() throws IOException {
...
+                    }
+                } catch (IOException ioe) {
+                    throw new RuntimeException(ioe);
+                };
+            });
+
+            final AtomicReference throwableReference =
+                new AtomicReference<Throwable>();
+            thr.setUncaughtExceptionHandler(
+                 new Thread.UncaughtExceptionHandler() {
```
Comment (GoeLin) [inline test/lib/jdk/test/lib/util/FileUtils.java:306]: Indentation is one off here.

Comment (luchenlin) [reply test/lib/jdk/test/lib/util/FileUtils.java]: fixed

**Review by GoeLin — APPROVED** [review]
LGTM

**Pull request closed without merging**